app/services/triple_eye_effable/cloud.rb

```diff
@@ -26,43 +26,49 @@ def initialize
 
     def create_resource(resourceable)
       response = self.class.post(base_url, body: request_body(resourceable), headers: headers)
-      resource_id, = parse_response(response)
-      create_description resourceable, resource_id
+      add_error(resourceable, response) and return unless response.success?
+
+      resource_id, data = parse_response(response)
+      resource_description = ResourceDescription.new(resource_id: resource_id)
+
+      populate_description resource_description, data
+      resourceable.resource_description = resource_description
     end
 
     def delete_resource(resourceable)
       return if resourceable.resource_description.nil?
 
       id = resourceable.resource_description.resource_id
-      self.class.delete("#{base_url}/#{id}", headers: headers)
+      response = self.class.delete("#{base_url}/#{id}", headers: headers)
+      add_error(resourceable, response) unless response.success?
     end
 
     def load_resource(resourceable)
       return if resourceable.resource_description.nil?
 
       resource_description = resourceable.resource_description
-      response = self.class.get("#{base_url}/#{resource_description.resource_id}")
-      resource_id, data = parse_response(response)
+      response = self.class.get("#{base_url}/#{resource_description.resource_id}", headers: headers)
```

Comment:
@blms - I believe the problem you encountered was a result of the API key not being sent on `GET` requests to the IIIF Cloud. Here we'll include the headers to make sure the API key is sent.

On the IIIF Cloud side, I've also added a [PR](https://github.com/performant-software/iiif-cloud/pull/25/files#diff-31edbd1987500bb45374e1cb41bde547203a0f2ab47362d1d5c6c1bc70bdf1d1R10) to check to make sure an API key is sent in order to authenticate.

Reply:
Excellent, thank you @dleadbetter! Let me know if you'd like me to try pulling both and running again.

Reply:
@blms - If you have a minute, that would be great. Thank you!

Reply:
Working now, thanks for resolving @dleadbetter!